services/src/ogc/wfs/request.rs

```diff
@@ -1,32 +1,42 @@
-use crate::ogc::util::{parse_bbox, parse_spatial_resolution_option, parse_time_option};
+#![allow(non_snake_case)]
+// TODO: remove this and use rename_all once utoipa supports this for `IntoParams`: https://github.com/juhaku/utoipa/issues/270
```

Comment:
uh, that's ugly. Is it possible to rename the parameter names with an utoipa attribute macro?

Reply:
unfortunately this is not possible. I was also surprised that this is not just a lint you can ignore for a line. Rust is really picky here.

Reply:
Could you at least put it on the struct(s)?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=e6bf3bac14f843e9040bcd11c3468bf4

And maybe you could comment on the issue. It looks like everything is fine there from the answer.